app/bundles/EmailBundle/Form/Type/EmailSendType.php

```diff
@@ -186,6 +195,30 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 );
             }
         }
+        if (!empty($options['with_immediately'])) {
+            if ($this->coreParametersHelper->getParameter('mailer_spool_type') == 'file') {
```

Comment:
Lets use `===` to avoid https://twitter.com/jan_linhart/status/786677685988753409